components/rowListScene.xml

```diff
@@ -1,7 +1,84 @@
 <?xml version="1.0" encoding="utf-8"?>
 <component name="rowListScene" extends="Scene" initialFocus="mainRowList">
   <script type="text/brightscript" uri="rowListScene.brs" />
+  <interface>
```

Comment:
Please fix the indentation (formatting) in this file.

Reply:
Done